models/stdp_triplet_connection.h

```diff
@@ -185,7 +187,7 @@ class STDPTripletConnection : public Connection< targetidentifierT >
   facilitate_( double_t w, double_t kplus, double_t ky )
   {
     double_t new_w = w + kplus * ( Aplus_ + Aplus_triplet_ * ky );
-    return new_w < Wmax_ ? new_w : Wmax_;
+    return new_w < std::abs( Wmax_ ) ? new_w : std::abs( Wmax_ );
```

Comment:
@weidel-p I agree with @heplesser and @sanjayankur31 that the handling of the sign issues for weight updates should be put into ``facilitate`` and ``depress`` respectively.